src/spans/awsSpan.js

```diff
@@ -11,7 +11,7 @@ import {
   stringifyAndPrune,
   isAwsService,
   parseErrorObject,
-  omitKeys,
+  omitKeys, getEventEntitySize,
```

Comment:
did you pretiffy?

Reply:
Mmm. It passed the tests. There was a bug in checks.sh and the circleci script. Fixing